ynput/OpenPype
# Unreal: New Publisher

## Brief description
Implementation of the new publisher for Unreal.

## Description
The implementation of the new publisher for Unreal. This PR includes the changes for all the existing creators to be compatible with the new publisher.
The basic creator has been split in two distinct creators:
- `UnrealAssetCreator`, works with assets in the Content Browser. 
- `UnrealActorCreator` that works with actors in the scene.

## Testing notes
You will need to have the new publisher enabled in the Experimental tools, in OP Settings. You will also need to import some assets in Unreal to test the creation of the instances.

1. Open the new publisher, from `Experimental tools -> New Publisher`.
2. Create at least one instance for each family.
3. Check if the created instance are collected correctly when closing and reopening the new publisher.
4. Try to publish the instances (note: there's no extractor for the `unrealStaticMesh`, so won't publish, but shouldn't give any error).
